conf/application.conf

```diff
@@ -123,6 +123,7 @@ features {
   stubContactPreferences.enabled = true
   contactPreferences.enabled = true
   languageSelector.enabled = true
+  changeWebsite.enabled = false
```

Comment:
Can you set this to `true` here for test/environment purposes. And could you raise an app-config-production PR to set it to `false` in Prod.